wizelineacademy/itesm-socioformador-ene-feb-2023-equipo-1
# Fixing details

#### What does this PR do?
(A brief explanation synthesizing the feature, bug or fix.)
- Adding newest changes onto Bot Script Branch and the frontend integration for uploading files.

#### Where should the reviewer start?
(Explain where the reviewer should start in order to review the whole addition or subtractions.)
- Check mostly bot response and information feeding as well as file uploading.

#### How should this be manually tested?
(List of steps to reproduce, corroborate or tests to run. Write this section clear enough so that external users can also follow it and test the fix.)
1. Test newest bot to see for answers.
2. Test questions from answers already given in the forums.
3. Test to see if files are uploaded.
4. Test to see if conversation is mantained by the bot.

#### Any background context you want to provide?
(Any information regarding the PR that the reviewer should know.)
None

#### What are the relevant tickets?
(Link to issues, related PR, JIRA issues, etc.)
Dealing with HU-17 Multi-Source Feeding for the Bot and general improvements in the algorithm.

#### Screenshots
(Screenshots of the feature if available.)
:camera:

#### Questions
(If available, to another developer or reviewer.)
:question:
